src/main/java/com/atlassian/bitbucket/jenkins/internal/model/TestResults.java

```diff
@@ -32,4 +32,9 @@ public int getFailed() {
     public int getIgnored() {
         return ignored;
     }
+
+    @Override
+    public String toString() {
+        return String.format("successful:%d,failed:%d,skipped:%d", successful, failed, ignored);
```

Comment:
Do we want to change `ignored` -> `skipped` in `TestResults.java` and in `BitbucketBuildFactoryTest.java` so that it's aligned with the Bitbucket side of things? I happy to leave it as is if ignored is what's used on the Jenkins side of things.

Reply:
I think _generally_ we try to follow Jenkins conventions in Jenkins, and translate them in Bitbucket. But we actually don't need this method, will remove